util/src/main/java/tech/pegasys/teku/util/config/Constants.java

```diff
@@ -115,7 +116,7 @@
   public static int EPOCHS_PER_RANDOM_SUBNET_SUBSCRIPTION = 256;
 
   // Sync
-  public static final UnsignedLong MAX_BLOCK_BY_RANGE_REQUEST_SIZE = UnsignedLong.valueOf(200);
+  public static final UnsignedLong MAX_BLOCK_BY_RANGE_REQUEST_SIZE = UnsignedLong.valueOf(1024);
```

Comment:
We don't want to increase this limit.  We should accept requests for up to 1024 blocks, but we only send 200 because we want to restrict how much work we have to do (the spec specifically allows us to not send all blocks back).

Reply:
ok I just figured when you said we could ask for more that you meant to increase the limit. i'll revert